basic/client/js/flux/index.js

```diff
@@ -107,6 +114,22 @@ const reducer = async (prevState, { type, payload }) => {
           return { ...prevState, error: err };
       }
     }
+    case DELETE_TODO_ACTION_TYPE: {
+      const url = api + '/' + payload.id;
+      try {
+        const resp = await fetch(url, { 
+          method: httpMethod.delete,
+          headers: headers
+        });
+        const index = prevState.todoList.findIndex(todo => todo.id === payload.id);
+        const nextTodoList = prevState.todoList.concat()
+        nextTodoList.splice(index, 1);
+        return {todoList: nextTodoList, error: null}
```

Comment:
超余談ですが、 `prevState` の property が `todoList` 以外に増えたときのことを見越して、以下のように書いておくとあとで困りにくいかと思います！
```suggestion
        return {...prevState, todoList: nextTodoList, error: null}
```

Reply:
ありがとうございます！
確かに変更があることを想定していなかったです！